test/forward/thrift.js

```diff
@@ -29,9 +29,14 @@ var someSpec = fs.readFileSync(
 var allocCluster = require('../lib/test-cluster.js');
 
 allocCluster.test('register and forward with thrift', {
-    size: 5
+    size: 5,
+    namedRemotes: ['echoRemote'],
+    remoteSpecs: {
```

Comment:
I feel that. But I think it would be wise to keep the top level property called remoteSpecs. What do you think?